packages/gluestick/src/config/webpack/webpack.config.js

```diff
@@ -110,28 +98,7 @@ module.exports = (gluestickConfig: GSConfig): WebpackConfig => {
       ],
     },
 
-    plugins: [
-      new webpack.LoaderOptionsPlugin({
-        test: /\.(scss|css)$/,
-        options: {
-          // A temporary workaround for `scss-loader`
-          // https://github.com/jtangelder/sass-loader/issues/298
-          output: {
-            path: outputPath,
-          },
-
-          postcss: [
-            autoprefixer({ browsers: 'last 2 version' }),
```

Comment:
These are such weird defaults. Why are CSS Custom Properties enabled in here? Why is there an opinionated autoprefixer setting?

Figure out what to do with these, but also move away from LoaderOptionsPlugin which is a Webpack 1 -> 2 crutch.